Apromore-Custom-Plugins/Log-Animation-Logic/src/main/java/org/apromore/service/loganimation/replay/Replayer.java

```diff
@@ -101,10 +101,10 @@ public String getProcessCheckingMsg() {
         return processCheckMessage;
     }
     
-    public AnimationLog replay(XLog log, String color) {
-        AnimationLog animationLog = new AnimationLog(log);
-        animationLog.setColor(color /*this.getLogColor()*/);
-        animationLog.setName(log.getAttributes().get("concept:name").toString());
+    public AnimationLog replay(XLog log, String name) {
+        AnimationLog animationLog = new AnimationLog(log, name);
+        //animationLog.setColor(color /*this.getLogColor()*/);
+        //animationLog.setName(log.getAttributes().get("concept:name").toString());
```

Comment:
Unless there's reason to believe these commented-out lines will ever return, may as well delete them for good.